DEVELOPING_OPAL/validate/src/test/java/org/opalj/fpcf/fixtures/xl/js/sandbox/JSAllocationWriteFieldFromJS.java

```diff
@@ -0,0 +1,31 @@
+package org.opalj.fpcf.fixtures.xl.js.sandbox;
```

Comment:
What are these commented out files and why are they part of this PR?

Reply:
files have been removed